board/amlogic/configs/m8b_m201_v1.h

```diff
@@ -288,8 +288,33 @@
     "loadbootenv=echo >>> Load Boot Script from mmc ${mmcbootdev}:1 <<<;ext4load mmc ${mmcbootdev}:1 0x11000000 /boot/uEnv.txt\0" \
     "importbootenv=echo >>> Importing environment from mmc ${mmcbootdev}:1 <<<;env import -t 0x11000000 ${filesize}\0" \
     "uenvbootcmd=ext4load mmc ${mmcbootdev}:1 ${loadaddr} /boot/${kernel_image};ext4load mmc ${mmcbootdev}:1 0x13000000 /boot/${ramdisk_image};bootm ${loadaddr}#${boardname} 0x13000000\0" \
+    "uenvbootcmd_append="\
+	"mmcinfo ${mmcbootdev}; run loadbootenv; run importbootenv; run getrootuuid; setenv bootargs root=UUID=${rootuuid} ${bootargs} ${enable_debug}; "\
```

Comment:
I think the duplicated bit that @dsd was referring to was this - if endlessboot is always run and selects between uenvbootcmd and uenvbootcmd_append, this line has already been run by the time _append is run.

Reply:
ok, I see it now. No reason, just a leftover when copying and pasting around.